src/Autocomplete/src/Form/WrappedEntityTypeAutocompleter.php

```diff
@@ -168,4 +169,9 @@ private function getEntityMetadata(): EntityMetadata
 
         return $this->entityMetadata;
     }
+
+    public function setOptions(array $options): void
```

Comment:
Maybe add some miniaml phpdoc type signature ? array<string, mixed> ? Or, as it must be query encoded maybe more specific ?

Reply:
The doc is present on the method in the interface. 
![CleanShot 2023-12-25 at 13 58 06@2x](https://github.com/symfony/ux/assets/80641364/08a66ed1-1075-42ef-a727-db62c5dc4c32)

At the end of day:
1. Interface is used (usually) as an argument type
2. IDE correctly detects the type of the argument

:P

Reply:
Oops... that's the problem of reviewing code in the gh interface ... sorry for that!